test.disabled/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -1354,6 +1377,42 @@ archived(Config) ->
     %% Made fresh in init_per_testcase
     escalus:story(Config, [{alice, 1}, {bob, 1}], F).
 
+
+message_with_stanzaid_and_archived(Config) ->
+    F = fun(Alice, Bob) ->
+        %% Archive must be empty.
+        %% Alice sends "OH, HAI!" to Bob.
+        escalus:send(Alice, escalus_stanza:chat_to(Bob, <<"OH, HAI!">>)),
+
+        %% Bob receives a message.
+        Msg = escalus:wait_for_stanza(Bob),
+        try
+        Arc_archived = exml_query:subelement(Msg, <<"archived">>),
```

Comment:
We currently don't use underscored names. They might be seen sometimes in older code but now e.g. it's `ArcStanzaID`, not `Arc_stanzaid`.